lib/index.js

```diff
@@ -82,10 +82,16 @@ class TemplateContext {
 class Template {
 	constructor(ast, options, templates) {
 		if (typeof ast === 'string') {
-			const parserOptions = {
+			let { babelOptions = {}, ...parserOptions } = options.parserOptions || {};
+
+			parserOptions = {
 				ecmaVersion: 2018,
 				requireConfigFile: false,
-				...(options.parserOptions || {}),
+				babelOptions: {
+					configFile: false,
```

Comment:
@futpib Is this mean to be `babelrc: false`?  It seems this package still trying to load config. 

```text
2021-04-08T13:05:39.0525727Z [13:05:39] → Parsing error: Cannot find module 'babel-plugin-dev-expression'
2021-04-08T13:05:39.0575536Z Require stack:
2021-04-08T13:05:39.0579518Z - /home/runner/work/eslint-plugin-unicorn/eslint-plugin-unicorn/node_modules/@babel/core/lib/config/files/plugins.js
2021-04-08T13:05:39.0614687Z - /home/runner/work/eslint-plugin-unicorn/eslint-plugin-unicorn/node_modules/@babel/core/lib/config/files/index.js
2021-04-08T13:05:39.0656581Z - /home/runner/work/eslint-plugin-unicorn/eslint-plugin-unicorn/node_modules/@babel/core/lib/index.js
2021-04-08T13:05:39.0658216Z - /home/runner/work/eslint-plugin-unicorn/eslint-plugin-unicorn/node_modules/@babel/eslint-parser/lib/index.js
2021-04-08T13:05:39.0662786Z - /home/runner/work/eslint-plugin-unicorn/eslint-plugin-unicorn/node_modules/eslint-template-visitor/lib/index.js
2021-04-08T13:05:39.0669120Z - /home/runner/work/eslint-plugin-unicorn/eslint-plugin-unicorn/node_modules/eslint-template-visitor/index.js
```

Reply:
Maybe it needs both `babelrc: false` and `configFile: false`. Can you create an issue so I can reproduce your problem if there is one?

Reply:
I don't know how to reproduce, here is the CI log https://github.com/fisker/eslint-plugin-unicorn/pull/266/checks?check_run_id=2296993058#step:5:956

Reply:
Or maybe because I dind't pass `configFile: false`?  Anyway, this line seems not right. https://github.com/futpib/eslint-template-visitor/blob/dfe46d502d960bd27c4b4b404fb71c89896c29d4/lib/index.js#L94

This should above `babelOptions:`